resources/icons/16px/bottom-row-three-tiles-view.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_279_19905)">
+<path d="M7 9C6.44772 9 6 9.44772 6 10L6 15C6 15.5523 6.44772 16 7 16H9C9.55228 16 10 15.5523 10 15V10C10 9.44772 9.55228 9 9 9H7Z" fill="black"/>
+<path d="M1 9C0.447716 9 2.42698e-07 9.44772 2.18557e-07 10L0 15C-2.41411e-08 15.5523 0.447715 16 1 16H3C3.55228 16 4 15.5523 4 15V10C4 9.44772 3.55228 9 3 9H1Z" fill="black"/>
+<path d="M13 9C12.4477 9 12 9.44772 12 10V15C12 15.5523 12.4477 16 13 16H15C15.5523 16 16 15.5523 16 15V10C16 9.44772 15.5523 9 15 9H13Z" fill="black"/>
+<path d="M1 0C0.447716 -2.41411e-08 2.42698e-07 0.447715 2.18557e-07 1L0 6C-2.41411e-08 6.55228 0.447715 7 1 7L15 7C15.5523 7 16 6.55229 16 6V1C16 0.447716 15.5523 6.36101e-07 15 6.1196e-07L1 0Z" fill="black"/>
+</g>
+<defs>
+<clipPath id="clip0_279_19905">
```

Comment:
these clip paths on many of the icons are causing rendering issues on the page, please remove all extraneous SVG markup from these files:

![image](https://github.com/palantir/blueprint/assets/723999/4ec4ab0b-6209-4989-b550-bccd411bbf55)